srcpkgs/python3-yfinance/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-yfinance'
+pkgname=python3-yfinance
+version=0.2.13b1
+revision=1
+build_style=python3-module
+makedepends="python3-setuptools"
+depends="python3 python3-appdirs python3-lxml python3-pandas python3-requests python3-numpy python3-multitasking python3-frozendict"
+checkdepends="${depends} python3-html5lib python3-BeautifulSoup4 python3-cryptography"
+short_desc="Download market data from Yahoo! Finance's API"
+maintainer="Jason Manley <jason@jasondavid.tv>"
+license="Apache-2.0"
+homepage="https://github.com/ranaroussi/yfinance"
+distfiles="${PYPI_SITE}/y/yfinance/yfinance-${version}.tar.gz"
+checksum=ac29e440e3f8721a7cc68ffbe63cd35e08db73e9cc27030da2fcc328fd885b54
+
+post_install() {
+	rm -f ${DESTDIR}/usr/bin/sample
```

Comment:
what is the best way to remove this so it doesn't cause conflicts with /usr/bin/sample ?